tests/ClosedXML.Report.Tests/XlsxTemplateTestsBase.cs

```diff
@@ -95,7 +97,9 @@ protected bool WorksheetsAreEqual(IXLWorksheet expected, IXLWorksheet actual, ou
             if (expected.Name != actual.Name)
                 messages.Add("Worksheet names differ");
 
-            if (expected.RangeUsed().RangeAddress.ToString() != actual.RangeUsed().RangeAddress.ToString())
+            var expectedRange = expected.RangeUsed();
+            var actualRange = actual.RangeUsed();
+            if (expectedRange != actualRange && expectedRange.RangeAddress.ToString() != actualRange.RangeAddress.ToString())
```

Comment:
I had been looking at this change for a minute trying to figure out how it could fix the failing test since `expectedRange` always differs from `actualRange` as they come from different workbooks... Only then I realized that it was a `NullReferenceException` you eliminated this way )

I think `if (expected.RangeUsed()?.RangeAddress?.ToString() != actual.RangeUsed()?.RangeAddress?.ToString())` would be more clear.

Reply:
Ok, thanks